Zotero/Scenes/General/Models/NoteEditorState.swift

```diff
@@ -64,6 +66,13 @@ struct NoteEditorState: ViewModelState {
         self.title = title
         changes = []
         self.openItemsCount = openItemsCount
+        displayTitle = generateDisplayTitle()
+    }
+
+    func generateDisplayTitle() -> String? {
```

Comment:
Should tab display title for a note consist of the "title" in this context? The "title" in note editor is parent's item display title and item type. So when viewing tabs you'd see "Parent title / actual note title", but it would be too big, so you probably wouldn't see much.

I'd do the following: 
1. Rename `displayTitle` to something like `sceneTitle` or `tabTitle` because it's a bit confusing. The `displayTitle` is not really displayed anywhere in the app, the `title` is displayed in navigation bar when available.
2. Don't generate `displayTitle` here, pass it from `RItem.displayTitle`, same as `title` is being passed. 
3. When the note updates, the `displayTitle` updates too, so you could update `displayTitle` on each save in `NoteEditorActionHandler.save`
4. Use only `RItem.displayTitle` of the note item, don't include parent item, it's too long anyway.

Reply:
Since this string is used for the current activity, as well as for the scene title in the app switcher, I picked `activityTitle` as a more apt name. Check https://github.com/zotero/zotero-ios/pull/829/commits/635fbde25259baa27e62e1b92b66f1f9e2fc8d01
Also made the same name change for `PDFReaderState` in https://github.com/zotero/zotero-ios/pull/829/commits/631b1459dd13e0c8386c9dc7f1245a67902cecf6